From 5ee58e69654ffd86bb4f050263de0c9da75fa28f Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Thu, 9 May 2019 00:05:02 +0000
Subject: [PATCH] OS-7794 Update Intel Microcode to Q1 QSR Reviewed by: John
 Levon <john.levon@joyent.com> Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: Jerry Jelinek
 <jerry.jelinek@joyent.com>

---
 manifest                             |  31 +++++++++++++++++++++++++++
 usr/src/data/ucode/README.ucode      |   4 ++--
 usr/src/data/ucode/intel/000206A7-02 | Bin 12288 -> 12288 bytes
 usr/src/data/ucode/intel/000206A7-10 | Bin 12288 -> 12288 bytes
 usr/src/data/ucode/intel/000306A9-02 | Bin 13312 -> 14336 bytes
 usr/src/data/ucode/intel/000306A9-10 | Bin 13312 -> 14336 bytes
 usr/src/data/ucode/intel/000306C3-02 | Bin 23552 -> 23552 bytes
 usr/src/data/ucode/intel/000306C3-10 | Bin 23552 -> 23552 bytes
 usr/src/data/ucode/intel/000306C3-20 | Bin 23552 -> 23552 bytes
 usr/src/data/ucode/intel/000306D4-40 | Bin 18432 -> 19456 bytes
 usr/src/data/ucode/intel/000306D4-80 | Bin 18432 -> 19456 bytes
 usr/src/data/ucode/intel/000306E4-01 | Bin 15360 -> 16384 bytes
 usr/src/data/ucode/intel/000306E4-04 | Bin 15360 -> 16384 bytes
 usr/src/data/ucode/intel/000306E4-08 | Bin 15360 -> 16384 bytes
 usr/src/data/ucode/intel/000306E4-20 | Bin 15360 -> 16384 bytes
 usr/src/data/ucode/intel/000306E4-40 | Bin 15360 -> 16384 bytes
 usr/src/data/ucode/intel/000306E4-80 | Bin 15360 -> 16384 bytes
 usr/src/data/ucode/intel/000306E7-01 | Bin 17408 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-04 | Bin 17408 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-08 | Bin 17408 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-20 | Bin 17408 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-40 | Bin 17408 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-80 | Bin 17408 -> 17408 bytes
 usr/src/data/ucode/intel/000306F2-01 | Bin 33792 -> 34816 bytes
 usr/src/data/ucode/intel/000306F2-02 | Bin 33792 -> 34816 bytes
 usr/src/data/ucode/intel/000306F2-04 | Bin 33792 -> 34816 bytes
 usr/src/data/ucode/intel/000306F2-08 | Bin 33792 -> 34816 bytes
 usr/src/data/ucode/intel/000306F2-20 | Bin 33792 -> 34816 bytes
 usr/src/data/ucode/intel/000306F2-40 | Bin 33792 -> 34816 bytes
 usr/src/data/ucode/intel/000306F4-80 | Bin 17408 -> 18432 bytes
 usr/src/data/ucode/intel/00040651-02 | Bin 22528 -> 21504 bytes
 usr/src/data/ucode/intel/00040651-10 | Bin 22528 -> 21504 bytes
 usr/src/data/ucode/intel/00040651-20 | Bin 22528 -> 21504 bytes
 usr/src/data/ucode/intel/00040651-40 | Bin 22528 -> 21504 bytes
 usr/src/data/ucode/intel/00040661-02 | Bin 25600 -> 25600 bytes
 usr/src/data/ucode/intel/00040661-10 | Bin 25600 -> 25600 bytes
 usr/src/data/ucode/intel/00040661-20 | Bin 25600 -> 25600 bytes
 usr/src/data/ucode/intel/00040671-02 | Bin 13312 -> 14336 bytes
 usr/src/data/ucode/intel/00040671-20 | Bin 13312 -> 14336 bytes
 usr/src/data/ucode/intel/000406E3-40 | Bin 99328 -> 100352 bytes
 usr/src/data/ucode/intel/000406E3-80 | Bin 99328 -> 100352 bytes
 usr/src/data/ucode/intel/000406F1-01 | Bin 28672 -> 30720 bytes
 usr/src/data/ucode/intel/000406F1-02 | Bin 28672 -> 30720 bytes
 usr/src/data/ucode/intel/000406F1-04 | Bin 28672 -> 30720 bytes
 usr/src/data/ucode/intel/000406F1-08 | Bin 28672 -> 30720 bytes
 usr/src/data/ucode/intel/000406F1-20 | Bin 28672 -> 30720 bytes
 usr/src/data/ucode/intel/000406F1-40 | Bin 28672 -> 30720 bytes
 usr/src/data/ucode/intel/000406F1-80 | Bin 28672 -> 30720 bytes
 usr/src/data/ucode/intel/00050654-01 | Bin 31744 -> 32768 bytes
 usr/src/data/ucode/intel/00050654-02 | Bin 31744 -> 32768 bytes
 usr/src/data/ucode/intel/00050654-04 | Bin 31744 -> 32768 bytes
 usr/src/data/ucode/intel/00050654-10 | Bin 31744 -> 32768 bytes
 usr/src/data/ucode/intel/00050654-20 | Bin 31744 -> 32768 bytes
 usr/src/data/ucode/intel/00050654-80 | Bin 31744 -> 32768 bytes
 usr/src/data/ucode/intel/00050655-01 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050655-02 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050655-04 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050655-10 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050655-20 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050655-80 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050656-01 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050656-02 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050656-04 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050656-08 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050656-10 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050656-20 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050656-80 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050657-01 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050657-02 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050657-04 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050657-08 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050657-10 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050657-20 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050657-80 | Bin 0 -> 47104 bytes
 usr/src/data/ucode/intel/00050662-10 | Bin 31744 -> 32768 bytes
 usr/src/data/ucode/intel/00050663-10 | Bin 22528 -> 24576 bytes
 usr/src/data/ucode/intel/00050664-10 | Bin 22528 -> 23552 bytes
 usr/src/data/ucode/intel/00050665-10 | Bin 18432 -> 19456 bytes
 usr/src/data/ucode/intel/000506C9-01 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000506C9-02 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000506CA-01 | Bin 14336 -> 15360 bytes
 usr/src/data/ucode/intel/000506CA-02 | Bin 14336 -> 15360 bytes
 usr/src/data/ucode/intel/000506E3-02 | Bin 99328 -> 100352 bytes
 usr/src/data/ucode/intel/000506E3-04 | Bin 99328 -> 100352 bytes
 usr/src/data/ucode/intel/000506E3-10 | Bin 99328 -> 100352 bytes
 usr/src/data/ucode/intel/000506E3-20 | Bin 99328 -> 100352 bytes
 usr/src/data/ucode/intel/000506F1-01 | Bin 10240 -> 11264 bytes
 usr/src/data/ucode/intel/000706A1-01 | Bin 73728 -> 73728 bytes
 usr/src/data/ucode/intel/000806E9-10 | Bin 0 -> 98304 bytes
 usr/src/data/ucode/intel/000806E9-40 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000806E9-80 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000806EA-40 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000806EA-80 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000806EB-10 | Bin 0 -> 98304 bytes
 usr/src/data/ucode/intel/000806EB-40 | Bin 0 -> 98304 bytes
 usr/src/data/ucode/intel/000806EB-80 | Bin 0 -> 98304 bytes
 usr/src/data/ucode/intel/000806EC-04 | Bin 0 -> 97280 bytes
 usr/src/data/ucode/intel/000806EC-10 | Bin 0 -> 97280 bytes
 usr/src/data/ucode/intel/000806EC-80 | Bin 0 -> 97280 bytes
 usr/src/data/ucode/intel/000906E9-02 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000906E9-08 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000906E9-20 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000906EA-02 | Bin 97280 -> 98304 bytes
 usr/src/data/ucode/intel/000906EA-20 | Bin 97280 -> 98304 bytes
 usr/src/data/ucode/intel/000906EB-02 | Bin 98304 -> 99328 bytes
 usr/src/data/ucode/intel/000906EC-02 | Bin 0 -> 98304 bytes
 usr/src/data/ucode/intel/000906EC-20 | Bin 0 -> 98304 bytes
 usr/src/data/ucode/intel/000906ED-02 | Bin 0 -> 97280 bytes
 usr/src/data/ucode/intel/000906ED-20 | Bin 0 -> 97280 bytes
 109 files changed, 33 insertions(+), 2 deletions(-)
 create mode 100644 usr/src/data/ucode/intel/00050655-01
 create mode 100644 usr/src/data/ucode/intel/00050655-02
 create mode 100644 usr/src/data/ucode/intel/00050655-04
 create mode 100644 usr/src/data/ucode/intel/00050655-10
 create mode 100644 usr/src/data/ucode/intel/00050655-20
 create mode 100644 usr/src/data/ucode/intel/00050655-80
 create mode 100644 usr/src/data/ucode/intel/00050656-01
 create mode 100644 usr/src/data/ucode/intel/00050656-02
 create mode 100644 usr/src/data/ucode/intel/00050656-04
 create mode 100644 usr/src/data/ucode/intel/00050656-08
 create mode 100644 usr/src/data/ucode/intel/00050656-10
 create mode 100644 usr/src/data/ucode/intel/00050656-20
 create mode 100644 usr/src/data/ucode/intel/00050656-80
 create mode 100644 usr/src/data/ucode/intel/00050657-01
 create mode 100644 usr/src/data/ucode/intel/00050657-02
 create mode 100644 usr/src/data/ucode/intel/00050657-04
 create mode 100644 usr/src/data/ucode/intel/00050657-08
 create mode 100644 usr/src/data/ucode/intel/00050657-10
 create mode 100644 usr/src/data/ucode/intel/00050657-20
 create mode 100644 usr/src/data/ucode/intel/00050657-80
 create mode 100644 usr/src/data/ucode/intel/000806E9-10
 create mode 100644 usr/src/data/ucode/intel/000806EB-10
 create mode 100644 usr/src/data/ucode/intel/000806EB-40
 create mode 100644 usr/src/data/ucode/intel/000806EB-80
 create mode 100644 usr/src/data/ucode/intel/000806EC-04
 create mode 100644 usr/src/data/ucode/intel/000806EC-10
 create mode 100644 usr/src/data/ucode/intel/000806EC-80
 create mode 100644 usr/src/data/ucode/intel/000906EC-02
 create mode 100644 usr/src/data/ucode/intel/000906EC-20
 create mode 100644 usr/src/data/ucode/intel/000906ED-02
 create mode 100644 usr/src/data/ucode/intel/000906ED-20

diff --git a/manifest b/manifest
index 477b6b916f..e3abfa9fc4 100644
--- a/manifest
+++ b/manifest
@@ -2167,6 +2167,26 @@ f platform/i86pc/ucode/GenuineIntel/00050654-04 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050654-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050654-20 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050654-80 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050655-01 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050655-02 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050655-04 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050655-10 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050655-20 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050655-80 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050656-01 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050656-02 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050656-04 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050656-08 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050656-10 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050656-20 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050656-80 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050657-01 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050657-02 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050657-04 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050657-08 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050657-10 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050657-20 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/00050657-80 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050662-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050663-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050664-10 0444 root sys
@@ -2182,16 +2202,27 @@ f platform/i86pc/ucode/GenuineIntel/000506E3-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506E3-20 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506F1-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000706A1-01 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000806E9-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000806E9-40 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000806E9-80 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000806EA-40 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000806EA-80 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000806EB-10 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000806EB-40 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000806EB-80 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000806EC-04 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000806EC-10 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000806EC-80 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000906E9-02 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000906E9-08 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000906E9-20 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000906EA-02 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000906EA-20 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000906EB-02 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000906EC-02 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000906EC-20 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000906ED-02 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000906ED-20 0444 root sys
 d proc 0755 root root
 d sbin 0755 root sys
 f sbin/autopush 0555 root bin
diff --git a/usr/src/data/ucode/README.ucode b/usr/src/data/ucode/README.ucode
index 964c984da4..24f4d5779c 100644
--- a/usr/src/data/ucode/README.ucode
+++ b/usr/src/data/ucode/README.ucode
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright (c) 2018, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 Microcode Management
@@ -30,4 +30,4 @@ the release that the microcode was obtained from here.
 AMD: Updated in March 2012 as part of illumos#2546. Exact revision
 unknown.
 
-Intel: Linux 20180807 release
+Intel: Linux 20190514 release
diff --git a/usr/src/data/ucode/intel/000206A7-02 b/usr/src/data/ucode/intel/000206A7-02
index e20cdbf957..844fc1fdb8 100644
Binary files a/usr/src/data/ucode/intel/000206A7-02 and b/usr/src/data/ucode/intel/000206A7-02 differ
diff --git a/usr/src/data/ucode/intel/000206A7-10 b/usr/src/data/ucode/intel/000206A7-10
index e20cdbf957..844fc1fdb8 100644
Binary files a/usr/src/data/ucode/intel/000206A7-10 and b/usr/src/data/ucode/intel/000206A7-10 differ
diff --git a/usr/src/data/ucode/intel/000306A9-02 b/usr/src/data/ucode/intel/000306A9-02
index f965ba53d5..896e410ca9 100644
Binary files a/usr/src/data/ucode/intel/000306A9-02 and b/usr/src/data/ucode/intel/000306A9-02 differ
diff --git a/usr/src/data/ucode/intel/000306A9-10 b/usr/src/data/ucode/intel/000306A9-10
index f965ba53d5..896e410ca9 100644
Binary files a/usr/src/data/ucode/intel/000306A9-10 and b/usr/src/data/ucode/intel/000306A9-10 differ
diff --git a/usr/src/data/ucode/intel/000306C3-02 b/usr/src/data/ucode/intel/000306C3-02
index 4dd6664e28..a6c51f0c29 100644
Binary files a/usr/src/data/ucode/intel/000306C3-02 and b/usr/src/data/ucode/intel/000306C3-02 differ
diff --git a/usr/src/data/ucode/intel/000306C3-10 b/usr/src/data/ucode/intel/000306C3-10
index 4dd6664e28..a6c51f0c29 100644
Binary files a/usr/src/data/ucode/intel/000306C3-10 and b/usr/src/data/ucode/intel/000306C3-10 differ
diff --git a/usr/src/data/ucode/intel/000306C3-20 b/usr/src/data/ucode/intel/000306C3-20
index 4dd6664e28..a6c51f0c29 100644
Binary files a/usr/src/data/ucode/intel/000306C3-20 and b/usr/src/data/ucode/intel/000306C3-20 differ
diff --git a/usr/src/data/ucode/intel/000306D4-40 b/usr/src/data/ucode/intel/000306D4-40
index 64d13670df..36c954116a 100644
Binary files a/usr/src/data/ucode/intel/000306D4-40 and b/usr/src/data/ucode/intel/000306D4-40 differ
diff --git a/usr/src/data/ucode/intel/000306D4-80 b/usr/src/data/ucode/intel/000306D4-80
index 64d13670df..36c954116a 100644
Binary files a/usr/src/data/ucode/intel/000306D4-80 and b/usr/src/data/ucode/intel/000306D4-80 differ
diff --git a/usr/src/data/ucode/intel/000306E4-01 b/usr/src/data/ucode/intel/000306E4-01
index c9792b7515..b60f219ea3 100644
Binary files a/usr/src/data/ucode/intel/000306E4-01 and b/usr/src/data/ucode/intel/000306E4-01 differ
diff --git a/usr/src/data/ucode/intel/000306E4-04 b/usr/src/data/ucode/intel/000306E4-04
index c9792b7515..b60f219ea3 100644
Binary files a/usr/src/data/ucode/intel/000306E4-04 and b/usr/src/data/ucode/intel/000306E4-04 differ
diff --git a/usr/src/data/ucode/intel/000306E4-08 b/usr/src/data/ucode/intel/000306E4-08
index c9792b7515..b60f219ea3 100644
Binary files a/usr/src/data/ucode/intel/000306E4-08 and b/usr/src/data/ucode/intel/000306E4-08 differ
diff --git a/usr/src/data/ucode/intel/000306E4-20 b/usr/src/data/ucode/intel/000306E4-20
index c9792b7515..b60f219ea3 100644
Binary files a/usr/src/data/ucode/intel/000306E4-20 and b/usr/src/data/ucode/intel/000306E4-20 differ
diff --git a/usr/src/data/ucode/intel/000306E4-40 b/usr/src/data/ucode/intel/000306E4-40
index c9792b7515..b60f219ea3 100644
Binary files a/usr/src/data/ucode/intel/000306E4-40 and b/usr/src/data/ucode/intel/000306E4-40 differ
diff --git a/usr/src/data/ucode/intel/000306E4-80 b/usr/src/data/ucode/intel/000306E4-80
index c9792b7515..b60f219ea3 100644
Binary files a/usr/src/data/ucode/intel/000306E4-80 and b/usr/src/data/ucode/intel/000306E4-80 differ
diff --git a/usr/src/data/ucode/intel/000306E7-01 b/usr/src/data/ucode/intel/000306E7-01
index 94415b1fa1..21a97de3e5 100644
Binary files a/usr/src/data/ucode/intel/000306E7-01 and b/usr/src/data/ucode/intel/000306E7-01 differ
diff --git a/usr/src/data/ucode/intel/000306E7-04 b/usr/src/data/ucode/intel/000306E7-04
index 94415b1fa1..21a97de3e5 100644
Binary files a/usr/src/data/ucode/intel/000306E7-04 and b/usr/src/data/ucode/intel/000306E7-04 differ
diff --git a/usr/src/data/ucode/intel/000306E7-08 b/usr/src/data/ucode/intel/000306E7-08
index 94415b1fa1..21a97de3e5 100644
Binary files a/usr/src/data/ucode/intel/000306E7-08 and b/usr/src/data/ucode/intel/000306E7-08 differ
diff --git a/usr/src/data/ucode/intel/000306E7-20 b/usr/src/data/ucode/intel/000306E7-20
index 94415b1fa1..21a97de3e5 100644
Binary files a/usr/src/data/ucode/intel/000306E7-20 and b/usr/src/data/ucode/intel/000306E7-20 differ
diff --git a/usr/src/data/ucode/intel/000306E7-40 b/usr/src/data/ucode/intel/000306E7-40
index 94415b1fa1..21a97de3e5 100644
Binary files a/usr/src/data/ucode/intel/000306E7-40 and b/usr/src/data/ucode/intel/000306E7-40 differ
diff --git a/usr/src/data/ucode/intel/000306E7-80 b/usr/src/data/ucode/intel/000306E7-80
index 94415b1fa1..21a97de3e5 100644
Binary files a/usr/src/data/ucode/intel/000306E7-80 and b/usr/src/data/ucode/intel/000306E7-80 differ
diff --git a/usr/src/data/ucode/intel/000306F2-01 b/usr/src/data/ucode/intel/000306F2-01
index d6f7d702fc..8445583922 100644
Binary files a/usr/src/data/ucode/intel/000306F2-01 and b/usr/src/data/ucode/intel/000306F2-01 differ
diff --git a/usr/src/data/ucode/intel/000306F2-02 b/usr/src/data/ucode/intel/000306F2-02
index d6f7d702fc..8445583922 100644
Binary files a/usr/src/data/ucode/intel/000306F2-02 and b/usr/src/data/ucode/intel/000306F2-02 differ
diff --git a/usr/src/data/ucode/intel/000306F2-04 b/usr/src/data/ucode/intel/000306F2-04
index d6f7d702fc..8445583922 100644
Binary files a/usr/src/data/ucode/intel/000306F2-04 and b/usr/src/data/ucode/intel/000306F2-04 differ
diff --git a/usr/src/data/ucode/intel/000306F2-08 b/usr/src/data/ucode/intel/000306F2-08
index d6f7d702fc..8445583922 100644
Binary files a/usr/src/data/ucode/intel/000306F2-08 and b/usr/src/data/ucode/intel/000306F2-08 differ
diff --git a/usr/src/data/ucode/intel/000306F2-20 b/usr/src/data/ucode/intel/000306F2-20
index d6f7d702fc..8445583922 100644
Binary files a/usr/src/data/ucode/intel/000306F2-20 and b/usr/src/data/ucode/intel/000306F2-20 differ
diff --git a/usr/src/data/ucode/intel/000306F2-40 b/usr/src/data/ucode/intel/000306F2-40
index d6f7d702fc..8445583922 100644
Binary files a/usr/src/data/ucode/intel/000306F2-40 and b/usr/src/data/ucode/intel/000306F2-40 differ
diff --git a/usr/src/data/ucode/intel/000306F4-80 b/usr/src/data/ucode/intel/000306F4-80
index f5967028bd..c373a6824d 100644
Binary files a/usr/src/data/ucode/intel/000306F4-80 and b/usr/src/data/ucode/intel/000306F4-80 differ
diff --git a/usr/src/data/ucode/intel/00040651-02 b/usr/src/data/ucode/intel/00040651-02
index 62a2dc9d13..f93f80134b 100644
Binary files a/usr/src/data/ucode/intel/00040651-02 and b/usr/src/data/ucode/intel/00040651-02 differ
diff --git a/usr/src/data/ucode/intel/00040651-10 b/usr/src/data/ucode/intel/00040651-10
index 62a2dc9d13..f93f80134b 100644
Binary files a/usr/src/data/ucode/intel/00040651-10 and b/usr/src/data/ucode/intel/00040651-10 differ
diff --git a/usr/src/data/ucode/intel/00040651-20 b/usr/src/data/ucode/intel/00040651-20
index 62a2dc9d13..f93f80134b 100644
Binary files a/usr/src/data/ucode/intel/00040651-20 and b/usr/src/data/ucode/intel/00040651-20 differ
diff --git a/usr/src/data/ucode/intel/00040651-40 b/usr/src/data/ucode/intel/00040651-40
index 62a2dc9d13..f93f80134b 100644
Binary files a/usr/src/data/ucode/intel/00040651-40 and b/usr/src/data/ucode/intel/00040651-40 differ
diff --git a/usr/src/data/ucode/intel/00040661-02 b/usr/src/data/ucode/intel/00040661-02
index ee019bb34b..d2d8fb5f8a 100644
Binary files a/usr/src/data/ucode/intel/00040661-02 and b/usr/src/data/ucode/intel/00040661-02 differ
diff --git a/usr/src/data/ucode/intel/00040661-10 b/usr/src/data/ucode/intel/00040661-10
index ee019bb34b..d2d8fb5f8a 100644
Binary files a/usr/src/data/ucode/intel/00040661-10 and b/usr/src/data/ucode/intel/00040661-10 differ
diff --git a/usr/src/data/ucode/intel/00040661-20 b/usr/src/data/ucode/intel/00040661-20
index ee019bb34b..d2d8fb5f8a 100644
Binary files a/usr/src/data/ucode/intel/00040661-20 and b/usr/src/data/ucode/intel/00040661-20 differ
diff --git a/usr/src/data/ucode/intel/00040671-02 b/usr/src/data/ucode/intel/00040671-02
index ed8a37ce73..2d9be24fd1 100644
Binary files a/usr/src/data/ucode/intel/00040671-02 and b/usr/src/data/ucode/intel/00040671-02 differ
diff --git a/usr/src/data/ucode/intel/00040671-20 b/usr/src/data/ucode/intel/00040671-20
index ed8a37ce73..2d9be24fd1 100644
Binary files a/usr/src/data/ucode/intel/00040671-20 and b/usr/src/data/ucode/intel/00040671-20 differ
diff --git a/usr/src/data/ucode/intel/000406E3-40 b/usr/src/data/ucode/intel/000406E3-40
index 96a70f69b7..8f7afa6efd 100644
Binary files a/usr/src/data/ucode/intel/000406E3-40 and b/usr/src/data/ucode/intel/000406E3-40 differ
diff --git a/usr/src/data/ucode/intel/000406E3-80 b/usr/src/data/ucode/intel/000406E3-80
index 96a70f69b7..8f7afa6efd 100644
Binary files a/usr/src/data/ucode/intel/000406E3-80 and b/usr/src/data/ucode/intel/000406E3-80 differ
diff --git a/usr/src/data/ucode/intel/000406F1-01 b/usr/src/data/ucode/intel/000406F1-01
index ff10f9a341..c524aec9c8 100644
Binary files a/usr/src/data/ucode/intel/000406F1-01 and b/usr/src/data/ucode/intel/000406F1-01 differ
diff --git a/usr/src/data/ucode/intel/000406F1-02 b/usr/src/data/ucode/intel/000406F1-02
index ff10f9a341..c524aec9c8 100644
Binary files a/usr/src/data/ucode/intel/000406F1-02 and b/usr/src/data/ucode/intel/000406F1-02 differ
diff --git a/usr/src/data/ucode/intel/000406F1-04 b/usr/src/data/ucode/intel/000406F1-04
index ff10f9a341..c524aec9c8 100644
Binary files a/usr/src/data/ucode/intel/000406F1-04 and b/usr/src/data/ucode/intel/000406F1-04 differ
diff --git a/usr/src/data/ucode/intel/000406F1-08 b/usr/src/data/ucode/intel/000406F1-08
index ff10f9a341..c524aec9c8 100644
Binary files a/usr/src/data/ucode/intel/000406F1-08 and b/usr/src/data/ucode/intel/000406F1-08 differ
diff --git a/usr/src/data/ucode/intel/000406F1-20 b/usr/src/data/ucode/intel/000406F1-20
index ff10f9a341..c524aec9c8 100644
Binary files a/usr/src/data/ucode/intel/000406F1-20 and b/usr/src/data/ucode/intel/000406F1-20 differ
diff --git a/usr/src/data/ucode/intel/000406F1-40 b/usr/src/data/ucode/intel/000406F1-40
index ff10f9a341..c524aec9c8 100644
Binary files a/usr/src/data/ucode/intel/000406F1-40 and b/usr/src/data/ucode/intel/000406F1-40 differ
diff --git a/usr/src/data/ucode/intel/000406F1-80 b/usr/src/data/ucode/intel/000406F1-80
index ff10f9a341..c524aec9c8 100644
Binary files a/usr/src/data/ucode/intel/000406F1-80 and b/usr/src/data/ucode/intel/000406F1-80 differ
diff --git a/usr/src/data/ucode/intel/00050654-01 b/usr/src/data/ucode/intel/00050654-01
index dda53a919d..c0f3959412 100644
Binary files a/usr/src/data/ucode/intel/00050654-01 and b/usr/src/data/ucode/intel/00050654-01 differ
diff --git a/usr/src/data/ucode/intel/00050654-02 b/usr/src/data/ucode/intel/00050654-02
index dda53a919d..c0f3959412 100644
Binary files a/usr/src/data/ucode/intel/00050654-02 and b/usr/src/data/ucode/intel/00050654-02 differ
diff --git a/usr/src/data/ucode/intel/00050654-04 b/usr/src/data/ucode/intel/00050654-04
index dda53a919d..c0f3959412 100644
Binary files a/usr/src/data/ucode/intel/00050654-04 and b/usr/src/data/ucode/intel/00050654-04 differ
diff --git a/usr/src/data/ucode/intel/00050654-10 b/usr/src/data/ucode/intel/00050654-10
index dda53a919d..c0f3959412 100644
Binary files a/usr/src/data/ucode/intel/00050654-10 and b/usr/src/data/ucode/intel/00050654-10 differ
diff --git a/usr/src/data/ucode/intel/00050654-20 b/usr/src/data/ucode/intel/00050654-20
index dda53a919d..c0f3959412 100644
Binary files a/usr/src/data/ucode/intel/00050654-20 and b/usr/src/data/ucode/intel/00050654-20 differ
diff --git a/usr/src/data/ucode/intel/00050654-80 b/usr/src/data/ucode/intel/00050654-80
index dda53a919d..c0f3959412 100644
Binary files a/usr/src/data/ucode/intel/00050654-80 and b/usr/src/data/ucode/intel/00050654-80 differ
diff --git a/usr/src/data/ucode/intel/00050655-01 b/usr/src/data/ucode/intel/00050655-01
new file mode 100644
index 0000000000..923b87da82
Binary files /dev/null and b/usr/src/data/ucode/intel/00050655-01 differ
diff --git a/usr/src/data/ucode/intel/00050655-02 b/usr/src/data/ucode/intel/00050655-02
new file mode 100644
index 0000000000..923b87da82
Binary files /dev/null and b/usr/src/data/ucode/intel/00050655-02 differ
diff --git a/usr/src/data/ucode/intel/00050655-04 b/usr/src/data/ucode/intel/00050655-04
new file mode 100644
index 0000000000..923b87da82
Binary files /dev/null and b/usr/src/data/ucode/intel/00050655-04 differ
diff --git a/usr/src/data/ucode/intel/00050655-10 b/usr/src/data/ucode/intel/00050655-10
new file mode 100644
index 0000000000..923b87da82
Binary files /dev/null and b/usr/src/data/ucode/intel/00050655-10 differ
diff --git a/usr/src/data/ucode/intel/00050655-20 b/usr/src/data/ucode/intel/00050655-20
new file mode 100644
index 0000000000..923b87da82
Binary files /dev/null and b/usr/src/data/ucode/intel/00050655-20 differ
diff --git a/usr/src/data/ucode/intel/00050655-80 b/usr/src/data/ucode/intel/00050655-80
new file mode 100644
index 0000000000..923b87da82
Binary files /dev/null and b/usr/src/data/ucode/intel/00050655-80 differ
diff --git a/usr/src/data/ucode/intel/00050656-01 b/usr/src/data/ucode/intel/00050656-01
new file mode 100644
index 0000000000..9aa09ed654
Binary files /dev/null and b/usr/src/data/ucode/intel/00050656-01 differ
diff --git a/usr/src/data/ucode/intel/00050656-02 b/usr/src/data/ucode/intel/00050656-02
new file mode 100644
index 0000000000..9aa09ed654
Binary files /dev/null and b/usr/src/data/ucode/intel/00050656-02 differ
diff --git a/usr/src/data/ucode/intel/00050656-04 b/usr/src/data/ucode/intel/00050656-04
new file mode 100644
index 0000000000..9aa09ed654
Binary files /dev/null and b/usr/src/data/ucode/intel/00050656-04 differ
diff --git a/usr/src/data/ucode/intel/00050656-08 b/usr/src/data/ucode/intel/00050656-08
new file mode 100644
index 0000000000..9aa09ed654
Binary files /dev/null and b/usr/src/data/ucode/intel/00050656-08 differ
diff --git a/usr/src/data/ucode/intel/00050656-10 b/usr/src/data/ucode/intel/00050656-10
new file mode 100644
index 0000000000..9aa09ed654
Binary files /dev/null and b/usr/src/data/ucode/intel/00050656-10 differ
diff --git a/usr/src/data/ucode/intel/00050656-20 b/usr/src/data/ucode/intel/00050656-20
new file mode 100644
index 0000000000..9aa09ed654
Binary files /dev/null and b/usr/src/data/ucode/intel/00050656-20 differ
diff --git a/usr/src/data/ucode/intel/00050656-80 b/usr/src/data/ucode/intel/00050656-80
new file mode 100644
index 0000000000..9aa09ed654
Binary files /dev/null and b/usr/src/data/ucode/intel/00050656-80 differ
diff --git a/usr/src/data/ucode/intel/00050657-01 b/usr/src/data/ucode/intel/00050657-01
new file mode 100644
index 0000000000..8884c4ec91
Binary files /dev/null and b/usr/src/data/ucode/intel/00050657-01 differ
diff --git a/usr/src/data/ucode/intel/00050657-02 b/usr/src/data/ucode/intel/00050657-02
new file mode 100644
index 0000000000..8884c4ec91
Binary files /dev/null and b/usr/src/data/ucode/intel/00050657-02 differ
diff --git a/usr/src/data/ucode/intel/00050657-04 b/usr/src/data/ucode/intel/00050657-04
new file mode 100644
index 0000000000..8884c4ec91
Binary files /dev/null and b/usr/src/data/ucode/intel/00050657-04 differ
diff --git a/usr/src/data/ucode/intel/00050657-08 b/usr/src/data/ucode/intel/00050657-08
new file mode 100644
index 0000000000..8884c4ec91
Binary files /dev/null and b/usr/src/data/ucode/intel/00050657-08 differ
diff --git a/usr/src/data/ucode/intel/00050657-10 b/usr/src/data/ucode/intel/00050657-10
new file mode 100644
index 0000000000..8884c4ec91
Binary files /dev/null and b/usr/src/data/ucode/intel/00050657-10 differ
diff --git a/usr/src/data/ucode/intel/00050657-20 b/usr/src/data/ucode/intel/00050657-20
new file mode 100644
index 0000000000..8884c4ec91
Binary files /dev/null and b/usr/src/data/ucode/intel/00050657-20 differ
diff --git a/usr/src/data/ucode/intel/00050657-80 b/usr/src/data/ucode/intel/00050657-80
new file mode 100644
index 0000000000..8884c4ec91
Binary files /dev/null and b/usr/src/data/ucode/intel/00050657-80 differ
diff --git a/usr/src/data/ucode/intel/00050662-10 b/usr/src/data/ucode/intel/00050662-10
index 44c2a7f731..75fc81dbd1 100644
Binary files a/usr/src/data/ucode/intel/00050662-10 and b/usr/src/data/ucode/intel/00050662-10 differ
diff --git a/usr/src/data/ucode/intel/00050663-10 b/usr/src/data/ucode/intel/00050663-10
index bdf73e86c9..387f5a4e19 100644
Binary files a/usr/src/data/ucode/intel/00050663-10 and b/usr/src/data/ucode/intel/00050663-10 differ
diff --git a/usr/src/data/ucode/intel/00050664-10 b/usr/src/data/ucode/intel/00050664-10
index d2b33205f7..21a6db266d 100644
Binary files a/usr/src/data/ucode/intel/00050664-10 and b/usr/src/data/ucode/intel/00050664-10 differ
diff --git a/usr/src/data/ucode/intel/00050665-10 b/usr/src/data/ucode/intel/00050665-10
index 2b38ee5d70..2800d7c687 100644
Binary files a/usr/src/data/ucode/intel/00050665-10 and b/usr/src/data/ucode/intel/00050665-10 differ
diff --git a/usr/src/data/ucode/intel/000506C9-01 b/usr/src/data/ucode/intel/000506C9-01
index a9739ad0b5..b999fdb674 100644
Binary files a/usr/src/data/ucode/intel/000506C9-01 and b/usr/src/data/ucode/intel/000506C9-01 differ
diff --git a/usr/src/data/ucode/intel/000506C9-02 b/usr/src/data/ucode/intel/000506C9-02
index a9739ad0b5..b999fdb674 100644
Binary files a/usr/src/data/ucode/intel/000506C9-02 and b/usr/src/data/ucode/intel/000506C9-02 differ
diff --git a/usr/src/data/ucode/intel/000506CA-01 b/usr/src/data/ucode/intel/000506CA-01
index 792380734f..6aaf638f96 100644
Binary files a/usr/src/data/ucode/intel/000506CA-01 and b/usr/src/data/ucode/intel/000506CA-01 differ
diff --git a/usr/src/data/ucode/intel/000506CA-02 b/usr/src/data/ucode/intel/000506CA-02
index 792380734f..6aaf638f96 100644
Binary files a/usr/src/data/ucode/intel/000506CA-02 and b/usr/src/data/ucode/intel/000506CA-02 differ
diff --git a/usr/src/data/ucode/intel/000506E3-02 b/usr/src/data/ucode/intel/000506E3-02
index 6e2cd0f68d..3dfc610744 100644
Binary files a/usr/src/data/ucode/intel/000506E3-02 and b/usr/src/data/ucode/intel/000506E3-02 differ
diff --git a/usr/src/data/ucode/intel/000506E3-04 b/usr/src/data/ucode/intel/000506E3-04
index 6e2cd0f68d..3dfc610744 100644
Binary files a/usr/src/data/ucode/intel/000506E3-04 and b/usr/src/data/ucode/intel/000506E3-04 differ
diff --git a/usr/src/data/ucode/intel/000506E3-10 b/usr/src/data/ucode/intel/000506E3-10
index 6e2cd0f68d..3dfc610744 100644
Binary files a/usr/src/data/ucode/intel/000506E3-10 and b/usr/src/data/ucode/intel/000506E3-10 differ
diff --git a/usr/src/data/ucode/intel/000506E3-20 b/usr/src/data/ucode/intel/000506E3-20
index 6e2cd0f68d..3dfc610744 100644
Binary files a/usr/src/data/ucode/intel/000506E3-20 and b/usr/src/data/ucode/intel/000506E3-20 differ
diff --git a/usr/src/data/ucode/intel/000506F1-01 b/usr/src/data/ucode/intel/000506F1-01
index 83b441b9bc..a1e3d414b7 100644
Binary files a/usr/src/data/ucode/intel/000506F1-01 and b/usr/src/data/ucode/intel/000506F1-01 differ
diff --git a/usr/src/data/ucode/intel/000706A1-01 b/usr/src/data/ucode/intel/000706A1-01
index 6fe711f7ce..43a86f2be8 100644
Binary files a/usr/src/data/ucode/intel/000706A1-01 and b/usr/src/data/ucode/intel/000706A1-01 differ
diff --git a/usr/src/data/ucode/intel/000806E9-10 b/usr/src/data/ucode/intel/000806E9-10
new file mode 100644
index 0000000000..c7fe3a961e
Binary files /dev/null and b/usr/src/data/ucode/intel/000806E9-10 differ
diff --git a/usr/src/data/ucode/intel/000806E9-40 b/usr/src/data/ucode/intel/000806E9-40
index c03eb315a3..49aea22aee 100644
Binary files a/usr/src/data/ucode/intel/000806E9-40 and b/usr/src/data/ucode/intel/000806E9-40 differ
diff --git a/usr/src/data/ucode/intel/000806E9-80 b/usr/src/data/ucode/intel/000806E9-80
index c03eb315a3..49aea22aee 100644
Binary files a/usr/src/data/ucode/intel/000806E9-80 and b/usr/src/data/ucode/intel/000806E9-80 differ
diff --git a/usr/src/data/ucode/intel/000806EA-40 b/usr/src/data/ucode/intel/000806EA-40
index 641d07ef43..20ff87c8c8 100644
Binary files a/usr/src/data/ucode/intel/000806EA-40 and b/usr/src/data/ucode/intel/000806EA-40 differ
diff --git a/usr/src/data/ucode/intel/000806EA-80 b/usr/src/data/ucode/intel/000806EA-80
index 641d07ef43..20ff87c8c8 100644
Binary files a/usr/src/data/ucode/intel/000806EA-80 and b/usr/src/data/ucode/intel/000806EA-80 differ
diff --git a/usr/src/data/ucode/intel/000806EB-10 b/usr/src/data/ucode/intel/000806EB-10
new file mode 100644
index 0000000000..9eae11d13c
Binary files /dev/null and b/usr/src/data/ucode/intel/000806EB-10 differ
diff --git a/usr/src/data/ucode/intel/000806EB-40 b/usr/src/data/ucode/intel/000806EB-40
new file mode 100644
index 0000000000..9eae11d13c
Binary files /dev/null and b/usr/src/data/ucode/intel/000806EB-40 differ
diff --git a/usr/src/data/ucode/intel/000806EB-80 b/usr/src/data/ucode/intel/000806EB-80
new file mode 100644
index 0000000000..9eae11d13c
Binary files /dev/null and b/usr/src/data/ucode/intel/000806EB-80 differ
diff --git a/usr/src/data/ucode/intel/000806EC-04 b/usr/src/data/ucode/intel/000806EC-04
new file mode 100644
index 0000000000..b0458f3abc
Binary files /dev/null and b/usr/src/data/ucode/intel/000806EC-04 differ
diff --git a/usr/src/data/ucode/intel/000806EC-10 b/usr/src/data/ucode/intel/000806EC-10
new file mode 100644
index 0000000000..b0458f3abc
Binary files /dev/null and b/usr/src/data/ucode/intel/000806EC-10 differ
diff --git a/usr/src/data/ucode/intel/000806EC-80 b/usr/src/data/ucode/intel/000806EC-80
new file mode 100644
index 0000000000..b0458f3abc
Binary files /dev/null and b/usr/src/data/ucode/intel/000806EC-80 differ
diff --git a/usr/src/data/ucode/intel/000906E9-02 b/usr/src/data/ucode/intel/000906E9-02
index a1cc285d60..51202b3fca 100644
Binary files a/usr/src/data/ucode/intel/000906E9-02 and b/usr/src/data/ucode/intel/000906E9-02 differ
diff --git a/usr/src/data/ucode/intel/000906E9-08 b/usr/src/data/ucode/intel/000906E9-08
index a1cc285d60..51202b3fca 100644
Binary files a/usr/src/data/ucode/intel/000906E9-08 and b/usr/src/data/ucode/intel/000906E9-08 differ
diff --git a/usr/src/data/ucode/intel/000906E9-20 b/usr/src/data/ucode/intel/000906E9-20
index a1cc285d60..51202b3fca 100644
Binary files a/usr/src/data/ucode/intel/000906E9-20 and b/usr/src/data/ucode/intel/000906E9-20 differ
diff --git a/usr/src/data/ucode/intel/000906EA-02 b/usr/src/data/ucode/intel/000906EA-02
index f4475ceea7..68aafca7ed 100644
Binary files a/usr/src/data/ucode/intel/000906EA-02 and b/usr/src/data/ucode/intel/000906EA-02 differ
diff --git a/usr/src/data/ucode/intel/000906EA-20 b/usr/src/data/ucode/intel/000906EA-20
index f4475ceea7..68aafca7ed 100644
Binary files a/usr/src/data/ucode/intel/000906EA-20 and b/usr/src/data/ucode/intel/000906EA-20 differ
diff --git a/usr/src/data/ucode/intel/000906EB-02 b/usr/src/data/ucode/intel/000906EB-02
index e0c1b96628..17fddc8670 100644
Binary files a/usr/src/data/ucode/intel/000906EB-02 and b/usr/src/data/ucode/intel/000906EB-02 differ
diff --git a/usr/src/data/ucode/intel/000906EC-02 b/usr/src/data/ucode/intel/000906EC-02
new file mode 100644
index 0000000000..5feee15c02
Binary files /dev/null and b/usr/src/data/ucode/intel/000906EC-02 differ
diff --git a/usr/src/data/ucode/intel/000906EC-20 b/usr/src/data/ucode/intel/000906EC-20
new file mode 100644
index 0000000000..5feee15c02
Binary files /dev/null and b/usr/src/data/ucode/intel/000906EC-20 differ
diff --git a/usr/src/data/ucode/intel/000906ED-02 b/usr/src/data/ucode/intel/000906ED-02
new file mode 100644
index 0000000000..b438fbddcb
Binary files /dev/null and b/usr/src/data/ucode/intel/000906ED-02 differ
diff --git a/usr/src/data/ucode/intel/000906ED-20 b/usr/src/data/ucode/intel/000906ED-20
new file mode 100644
index 0000000000..b438fbddcb
Binary files /dev/null and b/usr/src/data/ucode/intel/000906ED-20 differ
-- 
2.21.0

